R/transformer.R

```diff
@@ -8,5 +8,28 @@
 #'   custom glue transformers and some common use cases.
 #' @export
 identity_transformer <- function(text, envir) {
-  eval(parse(text = text, keep.source = FALSE), envir)
+  if (requireNamespace("rlang", quiet = TRUE)) {
+    tryCatch(
+      expr <- parse(text = text, keep.source = FALSE),
+      error = function(err) {
+        rlang::abort(
+          "Failed to parse glue component",
+          parent = err,
+          call = NULL
```

Comment:
The only way I could see to get the correct glue function here would be to use some global state, which didn't seem worth it to me.

Reply:
Agreed